pkgs/tools/inputmethods/fcitx5/sources.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
What is the benefit over this file? There isn't one repo for all the projects and the version numbers are also all over the place. It would be easier to just put a simple shellscript instead of this file and the python script that just called nix-update for all of these packages.

Reply:
Oh well, I wasn't aware of the existence of nix-update. I'll do what you just suggested instead now I guess.

Reply:
Unfortunately, nix-prefetch (used by nix-update) is not working for me.

Reply:
Are you using the one from master? Mic fixed a bug in it a couple of days ago and it maybe isn't on unstable yet. After the fix it worked again for me.

Reply:
I tried the one from master and It's still not working. Error messages:
```
A probably-wrong output hash of zeroes has been used, yet it somehow still succeeded in building.
Please report an issue at: https://github.com/msteen/nix-prefetch/issues
```

Reply:
Even if nix-prefetch works, I don't want to rely on such badly maintained and unreliable tool. I'd rather use a centralized `source.json` instead.

Reply:
Which package did you try updating? work for me with fcitx5.

Reply:
fcitx5, the command I ran was `nix-update --version 5.0.4 fcitx5`

Reply:
Works for me.

Reply:
Then it must be `/etc/nix/nix.conf` that caused the problem. Anyway, I'm not gonna bother wasting time on nix-prefetch. While it might not be a good idea to cram the sources of a set of packages into one file in general, I think it is justified in my case, where all of the packages are owned by the same organization, maintained by the same guy, and usually released at the same time.